inc/class-customizer-guided-tour.php

```diff
@@ -0,0 +1,234 @@
+<?php
+/**
+ * Customizer Guided Tour Class
+ *
+ * @author   WooThemes
+ * @since    0.8.8
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! class_exists( 'Customizer_NUX_Guided_Tour' ) ) :
+
+	class Customizer_NUX_Guided_Tour {
+		/**
+		 * Setup class.
+		 *
+		 * @since 0.8.8
+		 */
+		public function __construct() {
+			add_action( 'admin_init', array( $this, 'customizer' ) );
+		}
+
+		/**
+		 * Customizer.
+		 *
+		 * @since 0.8.8
+		 */
+		public function customizer() {
+			global $pagenow;
+
+			// Always load the assets when in the customizer.
+			if ( 'customize.php' === $pagenow ) {
+				add_action( 'customize_controls_enqueue_scripts',      array( $this, 'customize_scripts' ) );
+				add_action( 'customize_controls_print_footer_scripts', array( $this, 'print_templates' ) );
+				if ( isset( $_GET['store-wpcom-nux'] ) ) {
+					$this->isStorefrontActivated();
+				}
+			}
+		}
+
+		public function isStorefrontActivated() {
+			$current_theme = wp_get_theme();
+
+			if ( 'Storefront' == $current_theme->get( 'Name' ) ) {
+				$redirect_url = add_query_arg( array(
+					'sf_guided_tour' => '1',
+					'sf_tasks' => 'homepage',
+					'theme' => $_GET['theme'],
+				), admin_url( '/customize.php' ) );
+
+				wp_redirect( $redirect_url );
+				exit;
+			}
+		}
+
+		/**
+		 * Customizer enqueues.
+		 *
+		 * @since 0.8.8
+		 */
+		public function customize_scripts() {
+			global $storefront_version;
+
+			wp_enqueue_style( 'wc-store-nux-tour', WC_API_Dev::$plugin_url . 'assets/css/admin/customizer.css', array(), $storefront_version, 'all' );
+
+			wp_enqueue_script( 'wc-store-nux-tour', WC_API_Dev::$plugin_url . 'assets/js/admin/customizer.js', array( 'jquery', 'wp-backbone' ), $storefront_version, true );
```

Comment:
I think we should probably use our own version string here for cache busting instead of Storefront's. When testing I had an issue loading a cached copy (where the JS code still referenced WP-API-Dev). It'll be better to have a string we can update if we need to make any changes to our JS in the future.